docs/sources/operators-guide/architecture/components/distributor.md

```diff
@@ -101,7 +101,7 @@ The distributor waits for a successful response from `n`/2 + 1 ingesters, where
 
 ## Load balancing across distributors
 
-We recommend randomly load balancing write requests across distributor instances.
+Randomly load balance write requests across distributor instances.
```

Comment:
Why would a user want to do this? (In either phrasing, it is often helpful to provide rationale.) How about "For better performance, ..." or "For more server stability, ..." (making up suggestions on the spot here)?